src/redux/actions/getActions.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+import { GET_PARTIES } from './types';
+
+const headers = {
+        "Content-Type": "application/json",
+        "authorization": `Bearer ${localStorage.getItem("token")}`
+}
+export const fetchParties = () => dispatch => {
+        return axios.get('https://isaac-politico-api-heroku.herokuapp.com/api/v2/parties',
+        {
+                headers: headers
+        })
```

Comment:
Decouple this into a helper function so that you can reuse it across all the actions. The function will take parameters like `method`, `payload/data`, `endpoint`. Let me if you need more insight on the same.